apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransaction.cs

```diff
@@ -58,6 +58,13 @@ public sealed class SpannerTransaction : DbTransaction, ISpannerTransaction
 
         private Session Session { get; }
 
+        // Note: We use seconds here to follow the convention set by DbCommand.CommandTimeout.
```

Comment:
I'd rather keep it consistent with Command, which is locked in to the name due to it being an override.